src/args/HarvestsQueryArgs.ts

```diff
@@ -0,0 +1,22 @@
+import { IsEnum, IsInt, IsOptional, Min } from 'class-validator';
+import { Harvest_OrderBy, OrderDirection } from '../graphql/generated/badger-dao';
+
+export class HarvestsQueryArgs {
```

Comment:
what do you think about `src/graphql/args/` ? or is this standard pathing for these

Reply:
Since we're going to move to be a bit more resource based (for endpoints), I think it'd make sense to create directories that map to the resources (ex. `/src/harvests`, `/src/events/`, etc.).

Under `/src/harvests`, we can create an `/args` folder for request arguments, keep the services + controller under that, etc.

What do you think?